repo/tuf_on_ci/build_repository.py

```diff
@@ -3,14 +3,97 @@
 """Command line tool to build a publishable TUF-on-CI repository"""
 
 import logging
+import os
+import subprocess
+from datetime import UTC, datetime, timedelta
+from urllib import parse
 
 import click
+from tuf.api.metadata import Root, Signed, Targets
 
 from tuf_on_ci._repository import CIRepository
+from tuf_on_ci._version import __version__
 
 logger = logging.getLogger(__name__)
 
 
+def _git(cmd: list[str]) -> subprocess.CompletedProcess:
+    cmd = [
+        "git",
+        "-c",
+        "user.name=tuf-on-ci",
+        "-c",
+        "user.email=41898282+github-actions[bot]@users.noreply.github.com",
+        *cmd,
+    ]
+    proc = subprocess.run(cmd, check=True, capture_output=True, text=True)
+    logger.debug("%s:\n%s", cmd, proc.stdout)
+    return proc
+
+
+def build_description(repo: CIRepository) -> str:
+    lines = [
+        "## Signers",
```

Comment:
This title is not great. Maybe _Repository state_ or nothing at all (considering that jekyll will add the project name on top anyway)